src/Evervault/Evervault.php

```diff
@@ -80,6 +80,18 @@ public function decrypt($data) {
         return $this->httpClient->decrypt($data);
     }
 
+    public function createClientSideDecryptToken($data, $expiry) {
+        if (!$data) {
+            throw new EvervaultError('Please provide a payload');
+        }
+
+        if ($expiry) {
+            $expiry = $expiry * 1000;
+        }
+        
+        return $this->httpClient->createToken("api:decrypt", $data, $expiry);
```

Comment:
The grant is invalid:

```suggestion
        return $this->httpClient->createToken("decrypt:api", $data, $expiry);
```